src/components/Annotation/Comment/EditableComment.tsx

```diff
@@ -27,13 +25,19 @@ export const EditableComment = (props: EditableCommentProps) => {
 
   const textarea = useRef<HTMLTextAreaElement>(null);
 
-  const [value, setValue] = useState(comment.value);
+  const [value, setValue] = useState<string | undefined>(
+    // Hack for now
+    comment.value.length > 0 && comment.value.charAt(0) === '{'
+      ? JSON.parse(comment.value)
+      : comment.value
+  );
+  const [renderType, setRenderType] = useState<'text' | 'quill'>('text');
```

Comment:
BTW, I think the cleanest way to handle this is to make use of W3C Web Anno's own terminology. Quill vs. plaintext would essentially represent different content types, right? A body can have a `format` field. Formats _should_ comply to MIME types. But aren't strictly required to, so we could make up our own. Or we could simple use `text/plain` vs `application/json`.

Reply:
I guess my reasoning here is that while quill uses a JSON object, so do other RTE frameworks.  If we replace or have a different option for the RTE than specifying JSON will not be enough to distinguish one framework from another.

Reply:
Agreed about `application/json` not being enough. How about the following: 
- We use `text/plain` for the text comments, i.e. stick to the standard in this case (and potentially have interop with other tools when annotations are exported)
- We use `quill` as the format for the Quill comments?

Reply:
Sounds good.  I suggest we use application/quill which seems more in line other format options.